### ISSUE TITLE
Nested VComponent Redraw is not working easily

### ISSUE THREAD

**Issue by sclee15** [pragmagic/karax#49]
I made a VComponent which has a child VComponent

when child vcomponent calls
markDirty(self)
redraw()

it doesn't redraw the child component.
I can see the change when the parent component gets update.

Thank you.

**Comment by Araq** [comment]
Please write a simple test program demonstrating the problem. Probably we need a mechanism that propagates the dirtyness to parents.

**Comment by sclee15** [comment]
That's right. When I pass the VComponent ref to child component and do following solves the problems
but it seems there are better ways.

markDirty(self)
markDirty(parent)
redraw()

**Comment by ephja** [comment]
@Araq Are you implying you can't redraw just a child component? I'd like to redraw one child but not the other. Redrawing a large canvas for example can be expensive.
